src/components/standalone-link/index.tsx

```diff
@@ -24,8 +25,9 @@ const StandaloneLink = ({
       className={classes}
       download={download}
       href={href}
-      target={openInNewTab ? '_blank' : '_self'}
+      onClick={onClick}
       rel={openInNewTab ? 'noreferrer noopener' : undefined}
+      target={openInNewTab ? '_blank' : '_self'}
```

Comment:
🗒️ This line didn't change, just alphabetized the props after adding the new `onClick` prop. 😊